java/plasma/test.sh

```diff
@@ -35,7 +36,7 @@ pushd ../../cpp
             -DCMAKE_CXX_FLAGS="-g -O3" \
             -DARROW_BUILD_TESTS=off \
             -DARROW_HDFS=on \
-            -DARROW_BOOST_USE_SHARED=off \
+            -DARROW_BOOST_USE_SHARED=on \
```

Comment:
I had to do this to make it compile with Ubuntu 16.04, since the boost static libraries that ship with it are compiled without -fPIC.